File: shoppingCartInterface/src/app/Model/ShoppingCartReceipt.ts
export class ShoppingCartItemWithTax{
    ProductId:number;
    Name: string;
    Description:string;
    Cost: number;
    Quantity:number;
    TotalCostPrice: number;
    BasicSalesTax: number;
    ImportDutyTax: number;
    TotalSalesTax: number;
}

export class ShoppingCartReceipt{
    TotalCostPrice: number;
    BasicSalesTax: number;
    ImportDutyTax: number;
    GrandTotal: number;
    CartItems: ShoppingCartItemWithTax[]
}

File: eCommerceWebAPI/ShoppingCartLib/Model/ShoppingCartReceipt.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ShoppingCartLib.Model
{
	public class ShoppingCartReceipt
	{
		public double TotalCostPrice { get; set; }
		public double BasicSalesTax { get; set; }
		public double ImportDutyTax { get; set; }
		public double GrandTotal { get; set; }
		public List<ShoppingCartItemWithTax> CartItems { get; set; }
	}
}
